index.js

```diff
@@ -129,7 +176,17 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(year,movie) {
+  let releasedYear = []
+  for (let i=0 ; i < movies.length; i++){
+    if (movies[i].year >= year ){
```

Comment:
I think the problem here is that the `movie` objects don't actually have a key of `year`. Rather they have a key of `released` which seems to include a year in this string.